composer.json

```diff
@@ -102,7 +102,9 @@
             "@phpstan",
             "@phpcs",
             "@phpmd",
-            "@test"
+            "@test",
+            "@behat-adfs",
+            "@behat-metadata"
```

Comment:
I enabled 2 behat test features that where easily brought back to life. Most important of which are the ADFS tests.